services/director-v2/src/simcore_service_director_v2/api/routes/dynamic_services.py

```diff
@@ -271,3 +277,37 @@ async def service_restart_containers(
         raise LegacyServiceIsNotSupportedError() from error
 
     return NoContentResponse()
+
+
+@router.patch(
+    "/projects/{project_id}/-/project-networks",
```

Comment:
MINOR: i think prefix ``project-`` is redundant. ``/projects/{project_id}/-/networks``

Reply:
This can also work.
The current name would help you identify from the resource name what you are changing. In this case a "project-network".

Reply:
so now I do have a question, what is the rationale with the ```/-/```?

Reply:
Since a specific project network is not being targetted for this operation but all of them for the provided project_id. `-` (dash) stands for all. You could use `*` but this one gets escaped. We've seen with PC that others use `-`.

Reply:
ok. then it would be nice to have a reference. I could not find much on the internet about that convention.

Reply:
or ``/projects/{project_id}/-/sub-net``

Reply:
Will go for `/projects/{project_id}/-/networks`